.bitrise/build-and-test.yml

```diff
@@ -0,0 +1,54 @@
+---
+format_version: '11'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: android
+workflows:
+  build-and-test:
+    description: |-
+      # Build the project and run the test (unit and ui).
+    steps:
+      - cache-pull@2: { }
+      - install-missing-android-tools@3:
+          inputs:
+            - gradlew_path: "./gradlew"
+      - gradle-runner@2:
+          title: Checking code documentation
+          inputs:
+            - gradlew_path: "./gradlew"
+            - gradle_task: detekt detektTest
+      - gradle-runner@2:
+          title: Running unit tests
+          inputs:
+            - gradlew_path: "./gradlew"
+            - gradle_task: testDebug
+      - script:
```

Comment:
This step is not needed & the `EXECUTE_INTEGRATION_TESTS` variable it not being used.
feel free to delete it.